poucave/app.py

```diff
@@ -19,6 +19,32 @@
 HTML_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "html")
 
 
+class Checks:
+    def __init__(self, conf):
+        self.all = []
+        for project, checks in conf["checks"].items():
+            for name, params in checks.items():
+                check = Check(project, name, **params)
+                self.all.append(check)
```

Comment:
Parsing out from `conf` seems a little odd, can we maybe make this a `classmethod` called `from_conf` or something?